frontend/packages/kubevirt-plugin/src/components/modals/scheduling-modals/dedicated-resources-modal/index.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { NodeModel } from '@console/internal/models';
+import { createModalLauncher, ModalComponentProps } from '@console/internal/components/factory';
+import { Firehose } from '@console/internal/components/utils';
+import { getName, getNamespace } from '@console/shared';
+import { getVMLikeModel } from '../../../../selectors/vm';
+import { VMLikeEntityKind } from '../../../../types/vmLike';
+import { DedicatedResourcesModal } from './dedicated-resources-modal';
+
+const FirehoseDedicatedResourcesModal: React.FC<FirehoseDedicatedResourcesModalProps> = (props) => {
```

Comment:
can we move it to a separate file which is not index. So it can be clearly seen what is exported out of this component by a quick glance